[2022-03-25 00:06:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 07:46:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 07:46:36] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2022-03-25 07:46:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 07:46:42] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2022-03-25 07:46:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 07:46:49] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2022-03-25 07:46:55] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"/占卜","message_id":-990881599,"message_seq":440191,"message_type":"group","post_type":"message","raw_message":"/占卜","self_id":3102833903,"sender":{"age":0,"area":"","card":"陆鱼鱼","level":"","nickname":"陆鱼鱼🐈🐾","role":"admin","sex":"unknown","title":"🐟","user_id":1482698137},"sub_type":"normal","time":1648165590,"user_id":1482698137}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 07:46:55] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2022-03-25 08:53:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:53:33] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2022-03-25 08:53:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:53:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:53:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:53:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:53:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:53:52] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"/占卜","message_id":1856562549,"message_seq":440208,"message_type":"group","post_type":"message","raw_message":"/占卜","self_id":3102833903,"sender":{"age":0,"area":"","card":"陆鱼鱼","level":"","nickname":"陆鱼鱼🐈🐾","role":"admin","sex":"unknown","title":"🐟","user_id":1482698137},"sub_type":"normal","time":1648169607,"user_id":1482698137}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:53:56] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3774,"PacketSent":3400,"PacketLost":0,"MessageReceived":310,"MessageSent":55,"LastMessageTime":1648169607,"DisconnectTimes":2,"LostTimes":0}},"time":1648169611}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:54:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:54:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:54:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:54:20] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3778,"PacketSent":3404,"PacketLost":0,"MessageReceived":310,"MessageSent":55,"LastMessageTime":1648169607,"DisconnectTimes":2,"LostTimes":0}},"time":1648169636}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:54:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:54:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:54:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:54:45] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3778,"PacketSent":3404,"PacketLost":0,"MessageReceived":310,"MessageSent":55,"LastMessageTime":1648169607,"DisconnectTimes":2,"LostTimes":0}},"time":1648169660}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:54:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:54:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:55:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:55:10] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3779,"PacketSent":3405,"PacketLost":0,"MessageReceived":310,"MessageSent":55,"LastMessageTime":1648169607,"DisconnectTimes":2,"LostTimes":0}},"time":1648169685}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:55:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:55:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:55:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:55:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:55:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:55:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:55:32] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"柔风1x🐘","message_id":430953955,"message_seq":440209,"message_type":"group","post_type":"message","raw_message":"柔风1x🐘","self_id":3102833903,"sender":{"age":0,"area":"","card":"Erdbee","level":"","nickname":"不要说话","role":"admin","sex":"unknown","title":"🍐","user_id":921584143},"sub_type":"normal","time":1648169707,"user_id":921584143}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:55:34] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3781,"PacketSent":3406,"PacketLost":0,"MessageReceived":311,"MessageSent":55,"LastMessageTime":1648169707,"DisconnectTimes":2,"LostTimes":0}},"time":1648169710}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:55:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:55:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:55:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:55:59] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3782,"PacketSent":3407,"PacketLost":0,"MessageReceived":311,"MessageSent":55,"LastMessageTime":1648169707,"DisconnectTimes":2,"LostTimes":0}},"time":1648169734}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:56:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:56:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:56:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:56:23] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3783,"PacketSent":3408,"PacketLost":0,"MessageReceived":311,"MessageSent":55,"LastMessageTime":1648169707,"DisconnectTimes":2,"LostTimes":0}},"time":1648169759}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:56:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:56:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:56:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:56:48] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3784,"PacketSent":3409,"PacketLost":0,"MessageReceived":311,"MessageSent":55,"LastMessageTime":1648169707,"DisconnectTimes":2,"LostTimes":0}},"time":1648169783}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:56:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:56:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:57:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:57:12] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3784,"PacketSent":3409,"PacketLost":0,"MessageReceived":311,"MessageSent":55,"LastMessageTime":1648169707,"DisconnectTimes":2,"LostTimes":0}},"time":1648169808}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:57:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:57:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:57:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:57:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:57:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:57:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:57:37] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3785,"PacketSent":3410,"PacketLost":0,"MessageReceived":311,"MessageSent":55,"LastMessageTime":1648169707,"DisconnectTimes":2,"LostTimes":0}},"time":1648169832}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:57:40] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"来了","message_id":2036882438,"message_seq":440210,"message_type":"group","post_type":"message","raw_message":"来了","self_id":3102833903,"sender":{"age":0,"area":"","card":"夜枕","level":"","nickname":"夜枕","role":"member","sex":"unknown","title":"gyz","user_id":961719612},"sub_type":"normal","time":1648169835,"user_id":961719612}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:57:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:57:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:57:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:58:02] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3788,"PacketSent":3412,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648169857}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:58:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:58:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:58:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:58:26] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3789,"PacketSent":3413,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648169882}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:58:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:58:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:58:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:58:51] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3790,"PacketSent":3414,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648169906}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:58:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:59:02] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:59:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:59:15] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3790,"PacketSent":3414,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648169931}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:59:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:59:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:59:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 08:59:40] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3791,"PacketSent":3415,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648169955}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 08:59:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 08:59:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 08:59:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:00:04] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3792,"PacketSent":3416,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648169980}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:00:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:00:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:00:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:00:29] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3793,"PacketSent":3417,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648170004}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:00:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:00:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:00:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:00:54] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3794,"PacketSent":3418,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648170029}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:00:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:01:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:01:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:01:18] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3796,"PacketSent":3420,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648170054}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:01:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:01:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:01:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:01:43] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3796,"PacketSent":3420,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648170078}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:01:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:01:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:02:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:02:07] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3797,"PacketSent":3421,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648170103}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:02:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:02:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:02:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:02:32] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3798,"PacketSent":3422,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648170127}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:02:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:02:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:02:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:02:56] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3799,"PacketSent":3423,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648170152}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:03:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:03:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:03:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:03:21] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3800,"PacketSent":3424,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648170176}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:03:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:03:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:03:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:03:46] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3800,"PacketSent":3424,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648170201}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:03:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:03:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:04:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:04:10] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3801,"PacketSent":3425,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648170226}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:04:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:04:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:04:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:04:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:04:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:04:35] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3802,"PacketSent":3426,"PacketLost":0,"MessageReceived":312,"MessageSent":55,"LastMessageTime":1648169835,"DisconnectTimes":2,"LostTimes":0}},"time":1648170250}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:04:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:04:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:04:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:04:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:04:47] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"/占卜","message_id":242183312,"message_seq":440211,"message_type":"group","post_type":"message","raw_message":"/占卜","self_id":3102833903,"sender":{"age":0,"area":"","card":"快打这个修五","level":"","nickname":"巴","role":"member","sex":"unknown","title":"猪油包包","user_id":751900443},"sub_type":"normal","time":1648170262,"user_id":751900443}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:04:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:04:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:04:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:04:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:04:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:04:59] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3805,"PacketSent":3428,"PacketLost":0,"MessageReceived":313,"MessageSent":55,"LastMessageTime":1648170263,"DisconnectTimes":2,"LostTimes":0}},"time":1648170275}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:05:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:05:01] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"/占卜","message_id":-1754907350,"message_seq":440212,"message_type":"group","post_type":"message","raw_message":"/占卜","self_id":3102833903,"sender":{"age":0,"area":"","card":"快打这个修五","level":"","nickname":"巴","role":"member","sex":"unknown","title":"猪油包包","user_id":751900443},"sub_type":"normal","time":1648170276,"user_id":751900443}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:05:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:05:07] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"[CQ:image,file=90e194c1b3827508e137bf682189692f.image,url=https://gchat.qpic.cn/gchatpic_new/751900443/675683939-2272913194-90E194C1B3827508E137BF682189692F/0?term=3,subType=1]","message_id":-530510404,"message_seq":440213,"message_type":"group","post_type":"message","raw_message":"[CQ:image,file=90e194c1b3827508e137bf682189692f.image,subType=1]","self_id":3102833903,"sender":{"age":0,"area":"","card":"快打这个修五","level":"","nickname":"巴","role":"member","sex":"unknown","title":"猪油包包","user_id":751900443},"sub_type":"normal","time":1648170282,"user_id":751900443}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:05:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:05:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:05:24] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3808,"PacketSent":3429,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170299}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:05:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:05:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:05:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:05:48] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3809,"PacketSent":3430,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170324}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:05:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:06:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:06:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:06:13] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3809,"PacketSent":3430,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170348}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:06:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:06:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:06:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:06:37] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3810,"PacketSent":3431,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170373}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:06:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:06:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:06:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:07:02] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3811,"PacketSent":3432,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170397}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:07:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:07:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:07:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:07:27] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3812,"PacketSent":3433,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170422}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:07:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:07:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:07:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:07:51] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3813,"PacketSent":3434,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170447}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:07:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:08:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:08:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:08:16] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3813,"PacketSent":3434,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170471}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:08:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:08:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:08:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:08:40] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3815,"PacketSent":3436,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170496}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:08:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:08:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:08:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:09:05] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3816,"PacketSent":3437,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170520}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:09:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:09:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:09:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:09:29] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3817,"PacketSent":3438,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170545}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:09:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:09:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:09:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:09:54] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3818,"PacketSent":3439,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170569}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:09:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:10:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:10:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:10:19] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3819,"PacketSent":3440,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170594}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:10:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:10:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:10:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:10:43] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3819,"PacketSent":3440,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170619}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:10:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:10:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:11:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:11:08] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3820,"PacketSent":3441,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170643}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:11:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:11:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:11:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:11:32] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3821,"PacketSent":3442,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170668}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:11:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:11:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:11:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:11:57] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3823,"PacketSent":3444,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170692}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:12:02] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:12:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:12:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:12:21] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3824,"PacketSent":3445,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170717}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:12:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:12:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:12:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:12:46] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3824,"PacketSent":3445,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170741}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:12:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:12:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:13:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:13:10] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3825,"PacketSent":3446,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170766}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:13:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:13:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:13:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:13:35] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3826,"PacketSent":3447,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170790}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:13:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:13:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:13:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:14:00] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3827,"PacketSent":3448,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170815}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:14:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:14:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:14:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:14:24] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3828,"PacketSent":3449,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170840}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:14:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:14:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:14:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:14:49] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3829,"PacketSent":3450,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170864}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:14:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:15:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:15:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:15:13] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3829,"PacketSent":3450,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170889}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:15:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:15:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:15:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:15:38] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3831,"PacketSent":3452,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170913}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:15:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:15:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:15:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:16:02] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3832,"PacketSent":3453,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170938}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:16:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:16:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:16:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:16:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:16:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:16:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:16:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:16:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:16:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:16:27] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3833,"PacketSent":3454,"PacketLost":0,"MessageReceived":315,"MessageSent":55,"LastMessageTime":1648170283,"DisconnectTimes":2,"LostTimes":0}},"time":1648170962}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:16:29] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"[CQ:image,file=013972700627abce0dce472a96b4490e.image,url=https://gchat.qpic.cn/gchatpic_new/1482698137/675683939-2637140507-013972700627ABCE0DCE472A96B4490E/0?term=3,subType=0]","message_id":2114295839,"message_seq":440214,"message_type":"group","post_type":"message","raw_message":"[CQ:image,file=013972700627abce0dce472a96b4490e.image,subType=0]","self_id":3102833903,"sender":{"age":0,"area":"","card":"陆鱼鱼","level":"","nickname":"陆鱼鱼🐈🐾","role":"admin","sex":"unknown","title":"🐟","user_id":1482698137},"sub_type":"normal","time":1648170964,"user_id":1482698137}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:16:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:16:32] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"[CQ:image,file=745487d5862a243a912f47b4d7b04115.image,url=https://gchat.qpic.cn/gchatpic_new/1482698137/675683939-2515407304-745487D5862A243A912F47B4D7B04115/0?term=3,subType=0]","message_id":151169161,"message_seq":440215,"message_type":"group","post_type":"message","raw_message":"[CQ:image,file=745487d5862a243a912f47b4d7b04115.image,subType=0]","self_id":3102833903,"sender":{"age":0,"area":"","card":"陆鱼鱼","level":"","nickname":"陆鱼鱼🐈🐾","role":"admin","sex":"unknown","title":"🐟","user_id":1482698137},"sub_type":"normal","time":1648170967,"user_id":1482698137}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:16:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:16:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:16:52] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3836,"PacketSent":3455,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648170987}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:16:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:17:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:17:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:17:16] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3836,"PacketSent":3455,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171012}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:17:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:17:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:17:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:17:41] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3837,"PacketSent":3456,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171036}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:17:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:17:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:17:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:18:05] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3838,"PacketSent":3457,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171061}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:18:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:18:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:18:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:18:30] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3839,"PacketSent":3458,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171085}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:18:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:18:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:18:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:18:54] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3841,"PacketSent":3460,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171110}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:18:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:19:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:19:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:19:19] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3842,"PacketSent":3461,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171134}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:19:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:19:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:19:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:19:44] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3842,"PacketSent":3461,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171159}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:19:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:19:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:20:02] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:20:08] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3843,"PacketSent":3462,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171184}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:20:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:20:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:20:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:20:33] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3844,"PacketSent":3463,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171208}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:20:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:20:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:20:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:20:57] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3845,"PacketSent":3464,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171233}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:21:02] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:21:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:21:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:21:22] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3846,"PacketSent":3465,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171257}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:21:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:21:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:21:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:21:46] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3846,"PacketSent":3465,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171282}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:21:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:21:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:22:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:22:11] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3847,"PacketSent":3466,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171306}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:22:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:22:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:22:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:22:35] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3849,"PacketSent":3468,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171331}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:22:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:22:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:22:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:23:00] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3850,"PacketSent":3469,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171355}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:23:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:23:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:23:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:23:25] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3851,"PacketSent":3470,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171380}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:23:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:23:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:23:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:23:49] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3851,"PacketSent":3470,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171405}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:23:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:24:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:24:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:24:14] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3852,"PacketSent":3471,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171429}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:24:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:24:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:24:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:24:38] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3853,"PacketSent":3472,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171454}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:24:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:24:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:24:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:25:03] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3854,"PacketSent":3473,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171478}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:25:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:25:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:25:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:25:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:25:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:25:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:25:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:25:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:25:27] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3855,"PacketSent":3474,"PacketLost":0,"MessageReceived":317,"MessageSent":55,"LastMessageTime":1648170968,"DisconnectTimes":2,"LostTimes":0}},"time":1648171503}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:25:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:25:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:25:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:25:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:25:34] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"打兔兔","message_id":-1878263501,"message_seq":440216,"message_type":"group","post_type":"message","raw_message":"打兔兔","self_id":3102833903,"sender":{"age":0,"area":"","card":"快打这个修五","level":"","nickname":"巴","role":"member","sex":"unknown","title":"猪油包包","user_id":751900443},"sub_type":"normal","time":1648171510,"user_id":751900443}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:25:37] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"好！","message_id":-418592347,"message_seq":440217,"message_type":"group","post_type":"message","raw_message":"好！","self_id":3102833903,"sender":{"age":0,"area":"","card":"快打这个修五","level":"","nickname":"巴","role":"member","sex":"unknown","title":"猪油包包","user_id":751900443},"sub_type":"normal","time":1648171512,"user_id":751900443}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:25:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:25:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:25:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:25:46] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"[CQ:image,file=c029d931a4eb6a1236e6cc142e1b2426.image,url=https://gchat.qpic.cn/gchatpic_new/1251230387/675683939-2533622308-C029D931A4EB6A1236E6CC142E1B2426/0?term=3,subType=0]","message_id":2008273972,"message_seq":440218,"message_type":"group","post_type":"message","raw_message":"[CQ:image,file=c029d931a4eb6a1236e6cc142e1b2426.image,subType=0]","self_id":3102833903,"sender":{"age":0,"area":"","card":"想冬眠","level":"","nickname":"伊","role":"member","sex":"unknown","title":"🐍","user_id":1251230387},"sub_type":"normal","time":1648171521,"user_id":1251230387}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:25:52] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3860,"PacketSent":3476,"PacketLost":0,"MessageReceived":320,"MessageSent":55,"LastMessageTime":1648171521,"DisconnectTimes":2,"LostTimes":0}},"time":1648171527}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:25:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:26:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:26:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:26:17] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3860,"PacketSent":3476,"PacketLost":0,"MessageReceived":320,"MessageSent":55,"LastMessageTime":1648171521,"DisconnectTimes":2,"LostTimes":0}},"time":1648171552}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:26:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:26:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:26:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:26:41] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3861,"PacketSent":3477,"PacketLost":0,"MessageReceived":320,"MessageSent":55,"LastMessageTime":1648171521,"DisconnectTimes":2,"LostTimes":0}},"time":1648171577}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:26:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:26:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:26:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:27:06] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3862,"PacketSent":3478,"PacketLost":0,"MessageReceived":320,"MessageSent":55,"LastMessageTime":1648171521,"DisconnectTimes":2,"LostTimes":0}},"time":1648171601}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:27:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:27:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:27:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:27:30] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3863,"PacketSent":3479,"PacketLost":0,"MessageReceived":320,"MessageSent":55,"LastMessageTime":1648171521,"DisconnectTimes":2,"LostTimes":0}},"time":1648171626}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:27:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:27:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:27:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:27:55] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3864,"PacketSent":3480,"PacketLost":0,"MessageReceived":320,"MessageSent":55,"LastMessageTime":1648171521,"DisconnectTimes":2,"LostTimes":0}},"time":1648171650}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:28:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:28:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:28:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:28:19] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3864,"PacketSent":3480,"PacketLost":0,"MessageReceived":320,"MessageSent":55,"LastMessageTime":1648171521,"DisconnectTimes":2,"LostTimes":0}},"time":1648171675}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:28:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:28:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:28:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:28:44] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3865,"PacketSent":3481,"PacketLost":0,"MessageReceived":320,"MessageSent":55,"LastMessageTime":1648171521,"DisconnectTimes":2,"LostTimes":0}},"time":1648171699}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:28:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:28:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:29:02] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:29:09] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3866,"PacketSent":3482,"PacketLost":0,"MessageReceived":320,"MessageSent":55,"LastMessageTime":1648171521,"DisconnectTimes":2,"LostTimes":0}},"time":1648171724}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:29:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:29:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:29:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:29:33] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3868,"PacketSent":3484,"PacketLost":0,"MessageReceived":320,"MessageSent":55,"LastMessageTime":1648171521,"DisconnectTimes":2,"LostTimes":0}},"time":1648171749}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:29:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:29:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:29:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:29:58] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3869,"PacketSent":3485,"PacketLost":0,"MessageReceived":320,"MessageSent":55,"LastMessageTime":1648171521,"DisconnectTimes":2,"LostTimes":0}},"time":1648171773}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:30:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:30:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:30:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:30:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:30:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:30:22] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3870,"PacketSent":3486,"PacketLost":0,"MessageReceived":320,"MessageSent":55,"LastMessageTime":1648171521,"DisconnectTimes":2,"LostTimes":0}},"time":1648171798}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:30:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:30:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:30:33] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"/luck","message_id":11855010,"message_seq":440219,"message_type":"group","post_type":"message","raw_message":"/luck","self_id":3102833903,"sender":{"age":0,"area":"","card":"Shuuka","level":"","nickname":"Shuuka","role":"member","sex":"unknown","title":"渣男","user_id":1712430536},"sub_type":"normal","time":1648171808,"user_id":1712430536}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:30:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:30:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:30:47] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3871,"PacketSent":3486,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648171822}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:30:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:30:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:31:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:31:11] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3872,"PacketSent":3487,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648171847}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:31:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:31:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:31:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:31:36] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3873,"PacketSent":3488,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648171871}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:31:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:31:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:31:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:32:01] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3874,"PacketSent":3489,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648171896}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:32:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:32:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:32:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:32:25] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3875,"PacketSent":3490,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648171921}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:32:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:32:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:32:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:32:50] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3875,"PacketSent":3490,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648171945}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:32:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:33:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:33:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:33:14] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3877,"PacketSent":3492,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648171970}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:33:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:33:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:33:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:33:39] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3878,"PacketSent":3493,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648171994}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:33:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:33:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:33:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:34:03] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3879,"PacketSent":3494,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648172019}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:34:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:34:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:34:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:34:28] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3880,"PacketSent":3495,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648172043}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:34:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:34:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:34:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:34:53] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3881,"PacketSent":3496,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648172068}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:34:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:35:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:35:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:35:17] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3881,"PacketSent":3496,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648172093}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:35:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:35:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:35:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:35:42] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3882,"PacketSent":3497,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648172117}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:35:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:35:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:36:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:36:06] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3883,"PacketSent":3498,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648172142}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:36:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:36:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:36:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:36:31] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3885,"PacketSent":3500,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648172166}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:36:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:36:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:36:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:36:55] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3886,"PacketSent":3501,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648172191}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:37:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:37:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-25 09:37:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-25 09:37:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-25 09:37:20] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3886,"PacketSent":3501,"PacketLost":0,"MessageReceived":321,"MessageSent":55,"LastMessageTime":1648171809,"DisconnectTimes":2,"LostTimes":0}},"time":1648172215}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上线 
[2022-03-25 09:37:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": read tcp 127.0.0.1:60244->127.0.0.1:8000: wsarecv: An existing connection was forcibly closed by the remote host. 将进行第 2 次重试 
[2022-03-25 09:37:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": read tcp 127.0.0.1:60249->127.0.0.1:8000: wsarecv: An existing connection was forcibly closed by the remote host. 将进行第 1 次重试 
[2022-03-25 10:00:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:00:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:00:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:00:14] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3956,"PacketSent":3557,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648173601}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:00:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:00:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:00:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:00:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:00:34] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3956,"PacketSent":3557,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648173621}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:00:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:00:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:00:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:01:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:01:04] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3958,"PacketSent":3559,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648173651}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:04:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:04:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:04:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:04:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:04:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:04:49] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3967,"PacketSent":3568,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648173876}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:04:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:04:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:04:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:05:02] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3967,"PacketSent":3568,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648173889}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:05:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:05:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:05:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:05:14] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3968,"PacketSent":3569,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648173902}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:05:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:05:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:05:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:05:27] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3968,"PacketSent":3569,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648173914}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:05:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:05:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:05:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:05:39] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3968,"PacketSent":3569,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648173927}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:05:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:05:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:05:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:05:52] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3969,"PacketSent":3570,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648173939}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:05:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:05:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:06:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:06:05] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3969,"PacketSent":3570,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648173952}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:06:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:06:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:06:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:06:17] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3970,"PacketSent":3571,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648173965}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:06:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:06:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:06:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:06:30] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3970,"PacketSent":3571,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648173977}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:06:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:06:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:06:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:06:43] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3970,"PacketSent":3571,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648173990}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:06:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:06:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:06:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:06:55] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3971,"PacketSent":3572,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174003}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:06:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:07:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:07:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:07:08] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3971,"PacketSent":3572,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174015}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:07:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:07:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:07:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:07:21] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3972,"PacketSent":3573,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174028}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:07:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:07:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:07:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:07:33] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3972,"PacketSent":3573,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174041}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:07:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:07:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:07:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:07:46] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3974,"PacketSent":3575,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174053}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:07:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:07:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:07:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:07:59] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3974,"PacketSent":3575,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174066}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:08:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:08:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:08:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:08:11] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3974,"PacketSent":3575,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174079}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:08:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:08:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:08:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:08:24] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3975,"PacketSent":3576,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174091}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:08:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:08:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:08:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:08:36] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3975,"PacketSent":3576,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174104}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:08:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:08:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:08:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:08:49] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3976,"PacketSent":3577,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174116}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:08:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:08:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:08:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:09:02] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3976,"PacketSent":3577,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174129}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:09:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:09:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:09:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:09:14] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3977,"PacketSent":3578,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174142}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:09:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:09:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:09:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:09:27] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3977,"PacketSent":3578,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174154}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:09:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:09:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:09:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:09:40] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3977,"PacketSent":3578,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174167}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:09:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:09:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:09:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:09:52] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3978,"PacketSent":3579,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174180}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:09:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:09:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:10:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:10:05] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3978,"PacketSent":3579,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174192}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:10:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:10:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:10:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:10:18] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3979,"PacketSent":3580,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174205}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:10:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:10:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:10:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:10:30] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3979,"PacketSent":3580,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174218}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:10:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:10:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:10:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:10:43] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3979,"PacketSent":3580,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174230}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:10:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:10:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:10:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:10:56] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3980,"PacketSent":3581,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174243}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:10:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:11:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:11:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:11:08] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3980,"PacketSent":3581,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174256}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:11:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:11:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:11:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:11:21] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3982,"PacketSent":3583,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174268}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:11:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:11:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:11:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:11:33] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3982,"PacketSent":3583,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174281}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:11:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:11:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:11:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:11:46] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3983,"PacketSent":3584,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174293}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:11:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:11:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:11:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:11:59] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3983,"PacketSent":3584,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174306}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:12:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:12:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 10:12:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-25 10:12:11] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":3983,"PacketSent":3584,"PacketLost":0,"MessageReceived":334,"MessageSent":56,"LastMessageTime":1648173479,"DisconnectTimes":2,"LostTimes":0}},"time":1648174319}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-25 10:12:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-25 10:12:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-25 19:30:27] [ERROR]: Protocol -> unexpected disconnect: read tcp 10.250.211.139:59628->101.91.37.175:8080: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. 
[2022-03-25 19:32:25] [ERROR]: Protocol -> unexpected disconnect: read tcp 192.168.255.10:55969->42.81.183.253:443: wsarecv: An established connection was aborted by the software in your host machine. 
[2022-03-25 19:49:01] [ERROR]: Protocol -> unexpected disconnect: read tcp 10.250.211.139:56567->101.91.37.242:80: wsarecv: An existing connection was forcibly closed by the remote host. 
[2022-03-25 19:50:29] [ERROR]: Protocol -> unexpected disconnect: read tcp 192.168.255.10:50802->119.147.190.138:14000: wsarecv: An established connection was aborted by the software in your host machine. 
[2022-03-25 20:48:50] [ERROR]: Protocol -> unexpected disconnect: read tcp 10.250.211.139:58122->49.7.253.15:443: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. 
[2022-03-25 20:50:49] [ERROR]: Protocol -> unexpected disconnect: read tcp 192.168.255.10:53364->117.62.242.81:8080: wsarecv: An established connection was aborted by the software in your host machine. 
[2022-03-25 22:33:48] [ERROR]: Protocol -> unexpected disconnect: read tcp 10.250.211.139:54703->49.7.253.215:80: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. 
[2022-03-25 22:35:48] [ERROR]: Protocol -> unexpected disconnect: read tcp 192.168.255.10:54897->119.147.190.142:443: wsarecv: An established connection was aborted by the software in your host machine. 
[2022-03-25 23:43:53] [ERROR]: Protocol -> unexpected disconnect: read tcp 10.250.211.139:60708->117.62.242.81:14000: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. 
[2022-03-25 23:45:53] [ERROR]: Protocol -> unexpected disconnect: read tcp 192.168.255.10:51660->183.47.98.25:8080: wsarecv: An established connection was aborted by the software in your host machine. 
